Assets/Scripts/System/WorkspaceSaveSystem.cs

```diff
@@ -2,31 +2,41 @@
 using DynamicPanels;
 using System;
 using System.IO;
+using Zenject;
+using static Assets.Scripts.System.SettingsSystem;
 
 namespace Assets.Scripts.System
 {
+
     public class WorkspaceSaveSystem
     {
-        public string saveFilePath => Application.dataPath + "/panel_layout.txt";
-
+        //Dependency
+        private SettingsSystem settingSystem;
+        
+        [Inject]
+        public void Construct(
+            SettingsSystem settingSystem)
+        {
+            this.settingSystem = settingSystem;
+        }
+                
         public void Save(DynamicPanelsCanvas canvas)
         {
             byte[] data = PanelSerialization.SerializeCanvasToArray(canvas);
             string dataString = Convert.ToBase64String(data);
-            File.WriteAllText(saveFilePath, dataString);
+            settingSystem.panelSize.Set(dataString);
         }
 
         public void Load(DynamicPanelsCanvas canvas)
         {
-            if(!File.Exists(saveFilePath))
+            if(settingSystem.panelSize.Get()=="")
```

Comment:
Code style: please add some spaces between "commands". 
`if (settingSystem.panelSize.Get() == "")`
Hint: use auto formatting to fix these simple code style errors before committing